assets/js/src/core/modules/element/hooks/use-element-context.ts

```diff
@@ -37,3 +37,16 @@ export const useElementContext = (): UseElementContextReturn => {
   trackError(new GeneralError(errorMessage))
   throw new Error(errorMessage)
 }
+
+export const useOptionalElementContext = (): UseElementContextReturn | null => {
```

Comment:
Please call this method within the `useElementContext()` method to avoid that we need to duplicate the logic.

Reply:
good point - modified it with https://github.com/pimcore/studio-ui-bundle/pull/847/commits/bf6abc6f1788f66aa0516f4ac889dd1e982b960f